rdflib/extras/infixowl.py

```diff
@@ -1560,7 +1581,10 @@ def _getExtent():  # noqa: N802
 
 class Callable:
     def __init__(self, anycallable):
-        self.__call__ = anycallable
+        self.__callfn__ = anycallable
+
+    def __call__(self, *args, **kwargs):
+        return self.__callfn__(*args, **kwargs)
```

Comment:
@gjhiggins I'm not entirely clear what the purpose of this change is, is it really needed? Does it make any real difference?

I also think it would be best to use `self._callfn` instead of `self.__callfn__`, as we should not use dunder methods for our own internal purposes as they later could be used for something else by python. It is probably not that serious, but unless there is a good reason to use `__callfn__` I would rather change it to `_callfn`.

Reply:
I will change `__callfn__` to `_callfn` and merge.

Reply:
Done now.